src/models/core/AdditionalHttpHeaders.ts

```diff
@@ -1,11 +1,21 @@
 
+/**
+ * AdditionalHttpHeaders allows users to specify additional values for specific HTTP headers.
+ *
+ * @public
+ **/
+export interface AdditionalHttpHeaders {
+  /** {@inheritDoc ClientSDKHeaderValues} */
+  'Client-SDK'?: ClientSDKHeaderValues
+}
+
 /**
  * Additional agents and their versions used to create the Answers experience. The information for these
  * agents is added to the Client-SDK HTTP header along with that of the ANSWERS_CORE agent.
  *
  * @public
- **/
-export interface CustomClientSdk {
+ */
+export interface ClientSDKHeaderValues {
```

Comment:
if we are dropping `Values` from AdditionalHttpHeaders, do we want to drop it here as well, and just have `ClientSDKHeader`?

Reply:
Here it actually is values though